source/rust_verify/src/lifetime_generate.rs

```diff
@@ -137,6 +139,12 @@ impl State {
         Self::id(&mut self.rename_count, &mut self.id_to_name, IdKind::Local, &raw_id, f)
     }
 
+    fn field<S: Into<String>>(&mut self, raw_id: S) -> Id {
+        let raw_id = raw_id.into();
+        let f = || raw_id.clone();
+        Self::id(&mut self.rename_count, &mut self.field_to_name, IdKind::Field, &raw_id, f)
```

Comment:
Will this store the name in `key_to_name` even for numeric fields for which we are not going to use the `rename_count` in `lifetime_emit`?

Reply:
Yes

Reply:
Why is the distinction in `lifetime_emit` necessary?

Reply:
I guess it's not necessary, now that I think about it. I guess the original reason I did it was because I didn't think it made sense to use IdKind::Local for field names.